server/dash/dashboards/neighborhood.py

```diff
@@ -30,59 +32,97 @@
 
 def populate_options():
     values = []
-    for i in df.sort_values('council_name').council_name.unique():
-        values.append(
-            {
-                'label': i,
-                'value': i
-            }
-        )
+    for i in df.sort_values("council_name").council_name.unique():
+        values.append({"label": i, "value": i})
     return values
 
 
 # Define callback to update graph
 @app.callback(
-    Output('graph1', 'figure'),
-    [Input("council_list", "value")]
+    Output("graph1", "figure"),
+    [Input("council_list", "value"), Input("data_type", "value")],
 )
-def update_figure(selected_council):
+def update_figure(selected_council, selected_timeframe):
+    """Creates comparison graph
+
+    Args:
+        selected_council:
+          A string representing the council selected from the drop down menu.
+        selected_timeframe:
+          An integer representing the number of days around which to compute the moving average.
+          This is selected with radio buttons at the bottom of the graph.
+
+    Returns:
+        fig:
```

Comment:
For "Returns", we don't need to specify the name of the returned variable. See the [example](https://google.github.io/styleguide/pyguide.html#383-functions-and-methods) in the style guide.